### PR TITLE
Added check for non-empty package.json files

### PR DESCRIPTION
**Pull request by axieax**
Issue:
- Newly created package.json files are not valid json, resulting in the json parser raising the following error:

package_error

Changes:
- Added an additional check to ensure package.json files are non-empty

### CODE REVIEW

File: lua/package-info/modules/core/show.lua [hunk 1]
```diff
@@ -48,9 +47,8 @@ local get_package_metadata = function(current_package_version, outdated_dependen
 end
 
 local set_virtual_text = function(dependencies, outdated_dependencies)
-    local is_file_package_json = check_if_package_json()
-
-    if not is_file_package_json then
+    -- may be an unnecessary check
```
Comment (vuki656) [inline lua/package-info/modules/core/show.lua:50]: This is needed due to how the flow goes.

1. `npm outdated` command is ran
2. Virtual text is set

If you leave the buffer before the `npm outdated` command finishes, it's going to put virtual text in any buffer you are in, so this prevents that.

File: lua/package-info/modules/core/show.lua [hunk 2]
```diff
@@ -12,16 +12,15 @@ local utils = require("package-info.utils")
 ---------------------------------- HELPERS ---------------------------------
 ----------------------------------------------------------------------------
 
--- Checks if opened buffer is package.json file
-local check_if_package_json = function()
+-- Checks if the open buffer refers to a non-empty package.json file
+local is_valid_package_json = function()
     local current_buffer_name = vim.api.nvim_buf_get_name(0)
```
Comment (vuki656) [inline lua/package-info/modules/core/show.lua:17]: Since the code is pretty simple and self-explanatory I don't think we need comments here.

**Review by vuki656 — COMMENTED** [review]
Looking good, thanks.

Just some code-style stuff.

File: lua/package-info/modules/core/show.lua (hunk 1 above)
Comment (axieax) [reply lua/package-info/modules/core/show.lua:50]: Ohhh makes sense now! Thanks!

**Comment by axieax** [comment]
I have updated my PR in light of these comments :))

**Comment by vuki656** [comment]
Seems like code style pipeline is still failing.

Take care of that, squash the commits into one, and then it can be merged.

Once again thanks for the contribution.

**Comment by axieax** [comment]
Hi I'm not sure why the code style pipeline is still failing. Can you identify the problem?

**Comment by vuki656** [comment]
It is the quotes in the import. They should be double quotes.

**Comment by vuki656** [comment]
There are conflicts from my changes. Ill take care of that and then merge.

**Comment by vuki656** [comment]
Done, thanks again.

**Comment by axieax** [comment]
Thanks! Looking forward to seeing how this plugin develops 🚀

**Pull request merged**